sdk/iot/hub/samples/README.md

```diff
@@ -46,16 +49,27 @@ For more details on X509 authentication, refer to [this online documentation](ht
 
 ## Samples
 
-The following section documents various samples. All of them use the [Eclipse Paho MQTT C client][Eclipse_Paho].
-For all of these samples, the following environment variables will need to be set:
+For all of these samples, the following environment variables will need to be set.
 
+**NOTE**: These need to be set before running the cmake commands.
+
+- `VCPKG_DEFAULT_TRIPLET`: The triplet created using the instructions [here][VCPKG_DIRECTIONS].
+- `VCPKG_ROOT`: The full path to the VCPKG directory used to generate the triplet.
 - `AZ_IOT_DEVICE_ID`: Your device id.
 - `AZ_IOT_HUB_HOSTNAME`: The hostname of your IoT Hub.
 - `AZ_IOT_DEVICE_X509_CERT_PEM_FILE`: The full path to your device cert (in `.pem` format) concatenated
  with its private key.
 - `AZ_IOT_DEVICE_X509_TRUST_PEM_FILE`: The full path to the trusted server cert (in `.pem` format). This is usually
 not needed on Mac or Linux but might be needed for Windows.
 
+Once these are set, you MUST compile the SDK with the `BUILD_PAHO_TRANSPORT` option turned on. For example, on the command
+line, it might look like the following:
+```bash
+cmake -DBUILD_PAHO_TRANSPORT=ON ..
```

Comment:
Not sure if this is the right place to add cmake descriptions: they are likely going to run out of sync.

Reply:
I understand it could get out of sync but in this case, if the cmake option gets changed, it should be done with a find all and replace all and this section would come up. I think it complete the flow here of getting off the ground.